README.md

```diff
@@ -27,6 +27,9 @@ The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SH
 
 The keywords "unspecified", "undefined", and "implementation-defined" are to be interpreted as described in the [rationale for the C99 standard][c99-unspecified].
 
+An implementation is not compliant if it fails to satisfy one or more of the MUST, REQUIRED, or SHALL requirements.
+An implementation is compliant if it satisfies all the MUST, REQUIRED, and SHALL requirements.
```

Comment:
I [still][1] think we want the “for the protocols it implements” wording from runtime-spec and a list of protocols contained within this spec (and it seemed like @philips [agreed][2]).  But as it stands, 15ebbebe is better than nothing.  I'm fine having this land and then filing a follow-up PR with the protocol list.

[1]: https://github.com/opencontainers/image-spec/pull/381#discussion_r82910974
[2]: https://github.com/opencontainers/image-spec/pull/381#issuecomment-253296312